plugins/pastefromgdocs/plugin.js

```diff
@@ -0,0 +1,47 @@
+﻿/**
+ * @license Copyright (c) 2003-2019, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+( function() {
+	CKEDITOR.plugins.add( 'pastefromgdocs', {
+		requires: 'pastetools',
+
+		init: function( editor ) {
+			var pasteToolsPath = CKEDITOR.plugins.getPath( 'pastetools' ),
+				path = this.path;
+
+			editor.pasteTools.register( {
+				filters: [
+					CKEDITOR.getUrl( pasteToolsPath + 'filter/common.js' ),
+					CKEDITOR.getUrl(  path + 'filter/default.js' )
+				],
+
+				canHandle: function( evt ) {
+					return evt.data.dataValue.match( /id=(\"|\')docs\-internal\-guid\-/ );
```

Comment:
Why there is used `String.match` not `Regexp.test`? This statement seems to determine a boolean state and a result of a match seems to not be important.
SO claims that using `test` is faster for such cases:
https://stackoverflow.com/questions/10940137/regex-test-v-s-string-match-to-know-if-a-string-matches-a-regular-expression

Reply:
TBH without reason – this logic was copied from the previous PR and I didn't see a need of changing it (as it was working). But yeah, I can refactor it later.